src/background/services/networkFee/NetworkFeeService.ts

```diff
@@ -1,46 +1,35 @@
 import { NetworkVMType } from '@avalabs/core-chains-sdk';
 import { JsonRpcBatchInternal } from '@avalabs/core-wallets-sdk';
-import { isSwimmer } from '@src/utils/isSwimmerNetwork';
 import { getProviderForNetwork } from '@src/utils/network/getProviderForNetwork';
 import { singleton } from 'tsyringe';
-import { FeeRate, NetworkFee, TransactionPriority } from './models';
+import { NetworkFee } from './models';
 import { NetworkWithCaipId } from '../network/models';
 import { ModuleManager } from '@src/background/vmModules/ModuleManager';
 
-const EVM_BASE_TIP = BigInt(5e8); // 0.5 Gwei
-const EVM_TIP_MODIFIERS: Record<TransactionPriority, bigint> = {
-  low: 1n, // use the base tip
-  medium: 4n, // 4x base
-  high: 6n, // 6x base
-};
-
 @singleton()
 export class NetworkFeeService {
   constructor(private moduleManager: ModuleManager) {}
 
   async getNetworkFee(network: NetworkWithCaipId): Promise<NetworkFee | null> {
+    const module = await this.moduleManager.loadModuleByNetwork(network);
```

Comment:
I think these `vmName` conditions can be removed entirely. `.getNetworkFee()` response has the same shape for all VM modules.